clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -4845,4 +4845,39 @@ def CIR_AtomicClearOp : CIR_Op<"atomic.clear"> {
   }];
 }
 
+def CIR_MemScopeKind : CIR_I32EnumAttr<"MemScopeKind", "memory scope kind", [
```

Comment:
Seems like you are referring to an old version of incubator, we use `CIR_SyncScopeKind` there, please make sure this PR reflects the most updated version here and elsewhere in the PR.